src/NzbDrone.Core/Download/Clients/DownloadStation/TorrentDownloadStation.cs

```diff
@@ -199,6 +200,11 @@ protected bool IsFinished(DownloadStationTask torrent)
             return torrent.Status == DownloadStationTaskStatus.Finished;
         }
 
+        protected bool IsCompleted(DownloadStationTask torrent)
+        {
+            return torrent.Status == DownloadStationTaskStatus.Seeding || IsFinished(torrent);
```

Comment:
Just a lazy question, probably could look it up. But I meant: Is there a status that designates that the the torrent is scheduled to seed when a slot becomes available. Most download clients have a separate status for that.